### ISSUE TITLE
Is it possible to only allow hex colors in variable declarations?

### ISSUE THREAD

**Issue by benjamincharity** [sasstools/sass-lint#987]
With SCSS-lint I was able to define rules that colors such as `red` or `#bada55` can **not** be used in a style declaration, but **can** be used in a variable declaration.  Is there a way to replicate that behavior with this tool? ([scss-lint rule](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorvariable))

### Desired functionality

```scss
// Disallowed
.foo {
  background: #C0FF33;
  color: red;
}

// Allowed
$my_background: #FA113D;
$my_color: mistyrose;

.bar {
  background-color: $my_background;
  color: $my_color;
}
```

What version of Sass Lint are you using? `1.10.1`

**Comment by jackw** [comment]
Unless I'm missing something I'm pretty sure you're looking for the `no-color-literals` rule @benjamincharity. If you were using scss-lint there's a [tool here](http://sasstools.github.io/make-sass-lint-config/) that will convert your config over to sass-lint.

**Comment by benjamincharity** [comment]
@jackw you are correct! I didn't realize that rule was disabled in our repo a while back as it was conflicting with Sass helper function we wrote.

Thanks for pointing me in the correct direction 👍
